jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/DeadlinesBaseTest.java

```diff
@@ -409,24 +409,24 @@ public void testDelayedEmailNotificationOnDeadlineTaskCompletedMultipleTasks() t
           ((InternalOrganizationalEntity) user).setId("Administrator");
           ba.add(user);
           assignments.setBusinessAdministrators(ba);
-          
+
           List<OrganizationalEntity> po = new ArrayList<OrganizationalEntity>();
           User user2 = TaskModelProvider.getFactory().newUser();
-          ((InternalOrganizationalEntity) user2).setId("Administrator");        
+          ((InternalOrganizationalEntity) user2).setId("Administrator");
           po.add(user2);
           assignments.setPotentialOwners(po);
-          
+
           task.setPeopleAssignments(assignments);
-          
+
           
           // create task 2
           reader = new InputStreamReader(getClass().getResourceAsStream(MvelFilePath.DeadlineWithNotification));
           InternalTask task2 = (InternalTask) TaskFactory.evalTask(reader, vars);          
-          ((InternalTaskData) task2.getTaskData()).setSkipable(true);          
+          ((InternalTaskData) task2.getTaskData()).setSkipable(true);
           task2.setPeopleAssignments(assignments);
-          
+
           taskService.addTask(task, new HashMap<String, Object>());
-          taskService.addTask(task2, new HashMap<String, Object>());
+          taskService.addTask(task2, new HashMap<String, Object>());  // TODO - the TSR is accessed by drools during afterCompletion
```

Comment:
Sorry, I can remove that - it was addressed in my changes to the  Synchronization. I will remove the wording.